### PR TITLE
[12.0][ADD] web_widget_daterange : this module adds widget daterange

### PR DESCRIPTION
**Pull request by AyoubZahid**
This widget allow user to select start and end date from a single picker .
( this is a backport from odoo 13.0 widget daterange)

### CODE REVIEW

File: web_widget_daterange/static/src/js/widget.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+            });
+
+            // Prevent bootstrap from focusing on modal (which breaks hours drop-down in firefox)
+            this.$pickerContainer.on('focusin.bs.modal', 'select', function (ev) {
+                ev.stopPropagation();
+            });
+        },
+    });
+    field_registry.add('daterange', FieldDateRange);
+});
```
Comment (etobella) [inline web_widget_daterange/static/src/js/widget.js]: New line

File: web_widget_daterange/views/assets.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+<odoo>
+    <template id="assets_common" name="web_widget_daterange assets" inherit_id="web.assets_common">
+        <xpath expr="." position="inside">
+            <link rel="stylesheet" type="text/scss" href="/web_widget_daterange/static/src/scss/daterangepicker.scss"/>
+            <link rel="stylesheet" href="/web_widget_daterange/static/lib/daterangepicker/daterangepicker.css"/>
+            <script type="text/javascript" src="/web_widget_daterange/static/lib/daterangepicker/daterangepicker.js"></script>
+            <script type="text/javascript" src="/web_widget_daterange/static/src/js/widget.js"></script>
+        </xpath>
+    </template>
+</odoo>
```
Comment (etobella) [inline web_widget_daterange/views/assets.xml]: New line

**Review by etobella — CHANGES_REQUESTED** [review]
Can you add a readme on  the module?
I tried to make it work with `hr.holidays` and an error is raised. Can you check?
`ValueError: Wrong value for hr.leave.request_date_from_period: '2020-06-11T10:10:00.000Z'`

**Comment by AyoubZahid** [comment]
> Can you add a readme on the module?

Done 👍

**Pull request closed without merging**